vars.php

```diff
@@ -16,6 +16,11 @@
     define('PHPCI_BIN_DIR', PHPCI_DIR . 'vendor/bin/');
 }
 
+// Define PHPCI_BUILD_ROOT_DIR
+if (!defined('PHPCI_BUILD_ROOT_DIR')) {
+    define('PHPCI_BUILD_ROOT_DIR', PHPCI_DIR . 'PHPCI/build/');
```

Comment:
I suggest to use ```DIRECTORY_SEPARATOR``` eg:

    define('PHPCI_BUILD_ROOT_DIR', PHPCI_DIR . 'PHPCI' . DIRECTORY_SEPARATOR . 'build' . DIRECTORY_SEPARATOR);